Cartfile.resolved

```diff
@@ -3,8 +3,8 @@ github "DaveWoodCom/XCGLogger" "7.0.0"
 github "Sidetalker/onepassword-app-extension" "68a9d932a6373ca045984fbf4fb9a25aee43284c"
 github "SnapKit/SnapKit" "5.0.1"
 github "SwiftyJSON/SwiftyJSON" "5.0.0"
-github "airbnb/lottie-ios" "531eb22d47a9439a2344fdaf96832189f2c6e925"
-github "brave/brave-rewards-ios" "092cd4e52d9d821c7186de941adc6be8126baf23"
+github "airbnb/lottie-ios" "860124f1e8fe2163a87c225a931132fc57324b26"
```

Comment:
Is this change to Lottie library intentional?

Reply:
When I updated carthage, it also updated. If we don't want it, I can remove it but it works fine.

Reply:
Probably best to double check what changed in the update on `airbnb/lottie-ios` side.

Also you can update a single lib by passing the name into carthage update like: `carthage update brave-rewards-ios <other options>`